tests/test_concurrency.py

```diff
@@ -257,3 +259,75 @@ def test_multiexception_api():
     assert sucsessful.done()
     assert sucsessful.result() == 2
     assert sucsessful.exception() is None
+
+
+def test_logged_condition():
+    cond = LoggedCondition('test', log_interval=.2)
+
+    progress = 0
+    executed = []
+
+    def wait_for_progress_to(progress_to):
+        cond.wait_for(lambda: progress_to <= progress, 'progress to %s', progress_to)
+        executed.append(progress_to)
+
+    with concurrent(wait_for_progress_to, 10), concurrent(wait_for_progress_to, 20), concurrent(wait_for_progress_to, 30):
+        with patch("easypy.concurrency._logger") as _logger:
+            sleep(1)
+
+        assert sum(c == call("%s - waiting for progress to %s", cond, 10) for c in _logger.debug.call_args_list) > 3
+        assert sum(c == call("%s - waiting for progress to %s", cond, 20) for c in _logger.debug.call_args_list) > 3
+        assert sum(c == call("%s - waiting for progress to %s", cond, 30) for c in _logger.debug.call_args_list) > 3
+        assert executed == []
+
+        with patch("easypy.concurrency._logger") as _logger:
```

Comment:
why not patch it for the entire test?

Reply:
Because I run different checks on each one I patch, so it's easier to not have the calls mixed.